pcs/lib/commands/qdevice.py

```diff
@@ -157,16 +164,41 @@ def qdevice_kill(lib_env: LibraryEnvironment, model):
     _service_kill(lib_env, qdevice_net.SERVICE_NAME)
 
 
+def qdevice_net_get_ca_certificate(lib_env: LibraryEnvironment) -> bytes:
+    """
+    get base64 encoded qnetd CA certificate
+    """
+    path = os.path.join(
+        settings.corosync_qdevice_net_server_certs_dir,
+        settings.corosync_qdevice_net_server_ca_file_name,
+    )
+    try:
+        with open(path, "rb") as cert_file:
+            return base64.b64encode(cert_file.read())
+    except OSError as e:
+        lib_env.report_processor.report(
+            reports.ReportItem.error(
+                reports.messages.FileIoError(
+                    file_type_codes.COROSYNC_QNETD_CA_CERT,
+                    RawFileError.ACTION_READ,
+                    format_os_error(e),
+                    path,
+                )
+            )
+        )
+        raise LibraryError() from e
+
+
 def qdevice_net_sign_certificate_request(
     lib_env: LibraryEnvironment,
-    certificate_request,
-    cluster_name,
-):
+    certificate_request: str,
+    cluster_name: str,
+) -> bytes:
```

Comment:
And here we return file content encoded as base64, so maybe we can return `str`
```suggestion
) -> str:
```